### ISSUE TITLE
Bug: mark SC failed in WCAG EM if a page has a failure instance for that critieria

### ISSUE THREAD

**Issue by shkeating** [shkeating/nano-a11y-audit#26]
we want a criteria to mark as failed if a page fails for that critieria
